osu.Game.Tests/Editing/Checks/CheckFewHitsoundsTest.cs

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Audio;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Edit.Checks;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Tests.Beatmaps;
+
+namespace osu.Game.Tests.Editing.Checks
+{
+    [TestFixture]
+    public class CheckFewHitsoundsTest
```

Comment:
As far as I can see the nested HO case has no test coverage; would like to see some if possible.

Reply:
`MockNestableHitObject` could probably be reused for that. I'm not entirely sure where to put it though. Any ideas? Also do mock classes generally go in the test or non-test projects?

Reply:
I think putting it in `osu.Game.Tests.Editing.Checks` for now might be ok. Could see it being in `osu.Game.Tests.Beatmaps` as well.